src/pages/a/index.astro

```diff
@@ -19,23 +19,34 @@ const searchResults =
         language: Astro.currentLocale || 'en',
     }).then((r) => r.slice(0, 10)));
 
-const resultsWithAnalysis =
-    searchResults && searchResults.length > 0
-        ? await e
-              .group(
-                  e.select(e.Analysis, (a) => ({
-                      filter: e.all(
-                          e.set(
-                              e.op(a.app.platform, '=', e.literal(e.Platform, platform)),
-                              e.op(a.type, '=', e.literal(e.AnalysisType, 'initial')),
-                              e.op(a.app.appId, 'in', e.set(...searchResults.map((r) => r.id))),
+let resultsWithAnalysis: {
```

Comment:
This change seems unrelated. Is this necessary?

Reply:
If you were looking into the CI failures, that seems to be due to https://github.com/tweaselORG/parse-tunes/issues/14. Not sure why there are no results for WhatsApp, though. But I was going to look into that as part of https://github.com/tweaselORG/platform/pull/18.

Reply:
No, when debugging I ran into the problem that if the database connection fails, astro only throws with an unresolved promise instead of the actual error. Hence this error handling, which I think is useful regardless.